src/views/ops/Template/components/Variable.vue

```diff
@@ -68,6 +68,9 @@ export default {
     variables: {
       handler(value) {
         if (value.length > 0 || this.initial) {
+          value.map((item) => {
+            item.default_value = item.text_default_value || item.select_default_value
+          })
           this.$emit('input', value)
         }
         if (value) {
```

Comment:
The changes between the exported variables object have been made to:

1. Initialize `default_value` with `item.text_default_value` or `select_default_value`, based on `value.length`.

This change is not significant because it adds more functionality that does not need a check.

There isn't anything else notable in these differences; however, you should confirm the rest of the source's code before making major revisions.